easybuild/easyconfigs/o/OrthoFinder/OrthoFinder-2.3.8-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'Tarball'
+
+name = 'OrthoFinder'
+version = '2.3.8'
```

Comment:
Please bump to latest 2.3.11, since 2.3.10 introduced fixes for Python 3?

See comments in #9820 + https://github.com/davidemms/OrthoFinder/releases